library/cmac.c

```diff
@@ -300,8 +299,10 @@ int mbedtls_cipher_cmac_update( mbedtls_cipher_context_t *ctx,
     /* If there is data left over that wasn't aligned to a block */
```

Comment:
I just noticed that in the `for` loop above the line `cmac_ctx->padding_flag = 0;` is always executed. Is this because `mbedtls_cipher_update()` modifies it?